templates/payment-options-re.json

```diff
@@ -0,0 +1,105 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "payment-options-re event schema",
+  "type": "object",
+  "properties": {
+    "body": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "insertedTimestamp": {
+          "type": "string",
+          "description": "Data inserimento evento su kafka",
+          "examples": ["2024-09-23T17:27:04.1696470"]
+        },
+        "eventType": {
+          "type": "string",
+          "enum": ["REQ","RES","INTERNAL"]
+        },
+        "organizationId": {
+          "type": "string",
+          "description": "Id dell’EC"
+        },
+        "stationId": {
+          "type": "string",
+          "description": "Id della stazione"
+        },
+        "pspId": {
+          "type": "string",
+          "description": "Id del PSP"
+        },
+        "brokerId": {
+          "type": "string",
+          "description": "Id del broker"
+        },
+        "iuv": {
```

Comment:
for iuv and notice could be use https://json-schema.org/understanding-json-schema/reference/regular_expressions ? right @jacopocarlini @aomegax @marilenapaldino87 ?